tests/puppeteer-utils.js

```diff
@@ -92,7 +92,16 @@ const urlParams = (params) => urlString = Object.keys(params).map(
 ).join('&')
 
 const installRepo = async (page) => {
-  await page.goto(installUrl());
+  console.log( 'BRANCH_NAME:', BRANCH_NAME, '; pull url:', installUrl() );
```

Comment:
Not essential, but you could write this in the form of line 22, which would make it look cleaner, i.e., 

```console.log(`BRANCH_NAME: ${BRANCH_NAME}; pull url: ${installUrl()}`)```

Reply:
This is true. I never think in template strings. May conflict with other changes elsewhere, though, so maybe that'll be a future improvement.